elsa/_deployment.py

```diff
@@ -10,29 +10,39 @@
     ':octocat:', ':gift_heart:', ':tada:', ':balloon:', ':cake:', ':rocket:',
 ]
 
+def run(cmd, **kwargs):
+    """Same as ``subprocess.run``, but checks the result by default"""
+    kwargs.setdefault('check', True)
+    return subprocess.run(cmd, **kwargs)
+
+
+def get_last_commit_info(format):
+    cmd = ['git', 'show', '--format', format, '--no-patch']
```

Comment:
Consider adding `--no-pager` just after `git`. The pager tends to make trouble on some environments and only sometimes.